security/webauthn/src/test/java/MySqlWebAuthnIT.java

```diff
@@ -0,0 +1,26 @@
+import io.quarkus.test.bootstrap.MySqlService;
+import io.quarkus.test.bootstrap.RestService;
+import io.quarkus.test.scenarios.QuarkusScenario;
+import io.quarkus.test.services.Container;
+import io.quarkus.test.services.QuarkusApplication;
+
+@QuarkusScenario
+public class MySqlWebAuthnIT extends AbstractWebAuthnTest {
+
+    private static final int MYSQL_PORT = 3306;
+
+    @Container(image = "${mysql.80.image}", port = MYSQL_PORT, expectedLog = "ready for connections")
+    static MySqlService database = new MySqlService();
+
+    @QuarkusApplication
+    static RestService app = new RestService().withProperties("mysql.properties")
```

Comment:
I've added following waiting which fixed native mode. I don't suggest it as fix, but I believe it provides you with valuable information - the issue is that Quarkus app starts before app is ready. Personally I'd try to compare this with other Hibernate Reactive tests and see what is difference (IMO there is none, but I only had quick look). I'll leave you there as I have my own tasks. Good luck.

```
.onPreStart(new Action() {
                @Override
                public void handle(Service service) {
                    try {
                        Thread.sleep(Duration.ofSeconds(5).toMillis());
                    } catch (InterruptedException e) {
                        throw new RuntimeException(e);
                    }
                }
            })
```

Reply:
I also tried with your code lines but it failed in my ocp instance with the same failures. Also I increased the seconds...